planet_explorer/gui/pe_filters.py

```diff
@@ -456,6 +458,32 @@ def upload_file(self):
             layer = QgsVectorLayer(filename, "")
             self.aoi_from_layer(layer)
 
+    def show_aoi_area_size(self):
+        """Displays the aoi area size in square kilometers."""
+
+        area_size_sqkm = self.calculate_aoi_area()
+
+        formatted_area_sq = "{:,}".format(round(area_size_sqkm, 2))
+
+        self.laAOISize.setText(f"Total AOI area (sqkm): {formatted_area_sq}")
+
+    def calculate_aoi_area(self):
+        """Calculate the current aoi area in square kilometers"""
+
+        geometry = self.aoi_as_4326_geom()
+        area = QgsDistanceArea()
+        area.setSourceCrs(
+            QgsCoordinateReferenceSystem("EPSG:4326"),
+            QgsProject.instance().transformContext(),
+        )
+        area.setEllipsoid(QgsProject.instance().ellipsoid())
+        geometry_area = area.measureArea(geometry)
+        geometry_area_sq = area.convertAreaMeasurement(
+            geometry_area, QgsUnitTypes.AreaSquareKilometers
+        )
+
+        return round(geometry_area_sq, 2)
```

Comment:
For very small areas, this will report 0.0. Not a requirement but it'd be nice if we did "relative" rounding.